src/mds/MDBalancer.cc

```diff
@@ -1226,7 +1226,7 @@ void MDBalancer::hit_dir(CDir *dir, int type, int who, double amount)
     }
 
     if (dir->is_auth() && !dir->is_ambiguous_auth()) {
-      if (!dir->is_rep() &&
+      if (!dir->can_rep() &&
```

Comment:
@david-z Shouldn't this be `dir->can_rep()` instead of `!dir->can_rep()`?

Reply:
yes, thanks for catching this issue.